test_average (__main__.TestExercice) ... ok
test_square (__main__.TestExercice) ... ok
test_square_root (__main__.TestExercice) ... ok
test_to_celsius (__main__.TestExercice) ... ok
test_to_degrees (__main__.TestExercice) ... FAIL
test_to_farenheit (__main__.TestExercice) ... ok
test_to_radians (__main__.TestExercice) ... ok

======================================================================
FAIL: test_to_degrees (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 86, in test_to_degrees
    'Mauvais calcul de radians -> degres'
AssertionError: Lists differ: [(49.48552326356128, 5.881047377154118, 29.[198 chars]998)] != [(79.0980174562859, 5.881047377154118, 29.6[197 chars]998)]

First differing element 0:
(49.48552326356128, 5.881047377154118, 29.612494192724625)
(79.0980174562859, 5.881047377154118, 29.612494192724625)

Diff is 699 characters long. Set self.maxDiff to None to see it. : Mauvais calcul de radians -> degres

----------------------------------------------------------------------
Ran 7 tests in 0.005s

FAILED (failures=1)
